rhc.gemspec

```diff
@@ -30,7 +30,6 @@ Gem::Specification.new do |s|
     sep
   ].join("\n")
 
-  s.add_dependency              'net-ssh',      '>= 2.0.11', '<= 2.9.2'
   s.add_dependency              'net-scp',      '>= 1.1.2'
   s.add_dependency              'net-ssh-multi','>= 1.2.0'
   s.add_dependency              'archive-tar-minitar'
```

Comment:
Didn't we want to lock down versions for all dependencies to the latest known working version? Or do we not mind handling locking them down once a version is released that breaks us?